.travis.sh

```diff
@@ -25,6 +25,17 @@ sudo rosdep init
 
 rosdep update --include-eol-distros
 
+# install latest cmake for melodic
+if [[ "$ROS_DISTRO" = "melodic" ]]; then
```

Comment:
If we have to manually upgrade the cmake version, we have to write this install instruction in the README.md to infrom all melodic users. To be honest, I am afraid that this manual patch will bring some inconvenience. So I think we can just compromise for melodic user, and let them keep using the old nlopt version.